### DISCUSSION TITLE
Sensylink CHT8305/8310/8315/8320 support

### DISCUSSION THREAD

**Discussion by divadiow** [arendst/Tasmota#22735] [Q&A] [answered]
I don't believe I'm seeing any mention of these sensors in any discussions or supported sensor lists. I2C just detects my 8320 as a INA219

![image](https://github.com/user-attachments/assets/78176cbf-ae6b-4b42-a858-59943b6ecdb3)

I2C:Address 0x44 (dec 68)

any plans to support?

**Comment by sfromis** [comment, marked as answer]
We don't know if "someone" has plans of creating and submitting such a driver. Most drivers are submitted by people wanting additional support.
